tensorboard/components/tensor_widget/tensor-widget.ts

```diff
@@ -14,6 +14,9 @@ limitations under the License.
 ==============================================================================*/
 
 import {TensorWidget, TensorWidgetOptions, TensorView} from './types';
+import {VERSION} from './version';
+
+export const version = VERSION;
```

Comment:
Any reason not to `export {VERSION} from './version'`, to preserve the
case of the identifier?

Reply:
Two reasons:
1. It feels more idiomatic to have the public `version` attribute in lower case. E.g., `d3.version`, `Plotly.js`, TensorFlow.js's `tf.version`, etc.
2. This is so that all the public methods and constants can be exposed under the same module (i.e., `tensor-widget.ts` in this case).

Reply:
Counter-example of your (1) is Angular2 (`import {VERSION} from '@angular/core'`) and lodash (only two I searched for).
I am confused about your (2). You still can have `export {VERSION} from './version';` and achieve what you said.

Reply:
I don't feel strongly about the case of this symbol. Exporting it directly from tensor-widget.ts now, under all upper-case.